figo/figo.py

```diff
@@ -510,12 +510,10 @@ def remove_account(self, account_or_account_id):
         :Parameters:
          - `account_or_account_id` - account to be removed or its ID
         """
-        if isinstance(account_or_account_id, STRING_TYPES):
-            self._request_with_exception("/rest/accounts/%s" % account_or_account_id,
-                                           method="DELETE")
-        else:
-            self._request_with_exception("/rest/accounts/%s" % account_or_account_id.account_id,
-                                           method="DELETE")
+        if isinstance(account_or_account_id, Account):
+            account_or_account_id = account_or_account_id.account_id
+
+        self._request_with_exception("/rest/accounts/%s" % account_or_account_id, method="DELETE")
```

Comment:
It is preferable to use `str.format()` instead of the legacy percent-formatting.